src/controllers/graphql/resolvers/reply/replyResolver.js

```diff
@@ -1,12 +1,66 @@
 const db = require('../../connectors');
 const Reply = db.sequelize.models.reply;
+const ReplyService = require('../../../../services/replyService');
+const {
+  AuthenticationError,
+  gql,
+  makeExecutableSchema
+} = require('apollo-server');
 
-module.exports = {
+const typeDefs = gql`
+  scalar Date
+
+  type Query {
+    getAllRepliesByPostId: [Reply]
+    # getReplyById(id: ID!): Reply
+  }
+
+  type Mutation {
+    # createReply(reply: Reply!): Reply
```

Comment:
may need a login mutation

Reply:
Please describe the args and output we need from this mutation.

Reply:
I didn't describe it because I figured you'd be more familiar with what reply data we're receiving back from steem

Reply:
Seems you already wrote it in `/graphql/types/reply/`

```
  type Reply {
    id: Int!
    commenter: User!
    post: Post!
    parent: Reply
    permLink: String!
    body: String!
    createdAt: Date!
    netVotes: Int!
    pendingPayoutValue: Float!
    children: Int!
    depth: Int!
  }
```

Reply:
Ok, so to clarify, we need mutations that the user can only do when logged in.  This is what you're referring to right. Will get this done next.

Reply:
No, I think we've miscommunicated haha. You were asking me to describe the args/output for the login mutation right? I thought you were talking about the createReply mutation.